jenkins/Jenkinsfile

```
@@ -1,3 +1,10 @@
+def secrets = [
+  [path: 'secret/jenkins/wikimedia', engineVersion: 2, secretValues: [
+    [envVar: 'WIKIMEDIA_ACCESS_TOKEN', vaultKey: 'access_token']]],
```

I would change the name to be more generic. Right now we are using WikiMedia, but we are moving to StadiaMaps (that's why access token is needed).

Maybe
```suggestion
  [path: 'secret/jenkins/maps', engineVersion: 2, secretValues: [
    [envVar: 'MAPS_ACCESS_TOKEN', vaultKey: 'access_token']]],
```
?

---

ok, sounds good